app/Service/AdditionalTrainingService.php

```diff
@@ -25,6 +45,11 @@ public function getAdditionalTrainingDetails($uuid)
                 'course_ending_year' => $additionalTraining->course_ending_year,
                 'duration_hrs' => $additionalTraining->duration_hrs,
             ];
-        });
+        })->toArray();
```

Comment:
If you want, you can extract the mapping instruction to a specific function.

You have examples of this into ``LanguageService`` or ``StudentBootcampDetailService`` files